packages/node/CHANGELOG.md

```diff
@@ -6,10 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
The fix isn't on node. Changelog should be something like `Changed: update modulos to match changes with node core`